lib/XmppPrebind.php

```diff
@@ -65,9 +65,21 @@ class XmppPrebind {
 
 	protected $mechanisms = array();
 
+	// the Bosh attributes for use in a client using this prebound session
+	protected $wait,
+			$requests,
```

Comment:
Can you split each variable to own definition?
```php
protected $wait;
protected $requests;
```

It would be easier to add a docblock (with link to documentation) for each in future.